lib/tabbed-dialog.jsx

```diff
@@ -30,8 +30,8 @@ export default React.createClass( {
 			<Dialog className="settings" {...dialog}>
 				<nav className="dialog-tabs theme-color-border">
 					<ul>
-						{tabs.map( tab =>
-							<li key={tab}>
+						{tabs.map( ( tab, key ) =>
+							<li key={key}>
```

Comment:
This is related to the comment above. I came in to fix the key issue here since it was relatively easy. It seems like most of the files follow the ideology that spaces are expensive and should be avoided.